component/authenticator/src/test/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtilTest.java

```diff
@@ -601,4 +605,48 @@ public IObjectFactory getObjectFactory() {
 
         return new PowerMockObjectFactory();
     }
+
+    @Test(description = "Test case for getProcessedClaimValue()", dataProvider = "processedClaimValueTestDataProvider")
+    public void testGetProcessedClaimValue(String claimURI, List<LocalClaim> localClaims, String claimValue,
+                                           String expectedClaimValue) throws Exception {
+
+        mockStatic(TOTPDataHolder.class);
+        when(totpDataHolder.getInstance()).thenReturn(totpDataHolder);
+        when(totpDataHolder.getClaimMetadataManagementService()).thenReturn(claimMetadataManagementService);
+        when(claimMetadataManagementService.getLocalClaims("testDomain")).thenReturn(localClaims);
+        assertEquals(TOTPUtil.getProcessedClaimValue(claimURI, claimValue,"testDomain"), expectedClaimValue);
+    }
+
+    @DataProvider(name = "processedClaimValueTestDataProvider")
+    public static Object[][] getProcessedClaimValueTestData() {
+
+        HashMap<String, String> claimProperties_1 = new HashMap<>();
+        claimProperties_1.put("EnableEncryption", "true");
+        List<LocalClaim> localClaims1 = new ArrayList<>();
+        LocalClaim localClaim1 = new LocalClaim("http://wso2.org/claims/identity/secretkey", null,
```

Comment:
`http://wso2.org/claims/identity/secretkey` -> isn't there any constant defined for this?

Reply:
Will be addressed in https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/162